rmw_connext_dynamic_cpp/src/functions.cpp

```diff
@@ -996,22 +996,22 @@ rmw_create_publisher(
       }
     }
     if (ddts) {
-      // Cannot unregister type here, in case another topic is using the same type.
+      // TODO Cannot unregister and free type here, in case another topic is using the same type.
```

Comment:
Just to check I'm understanding this: we currently don't have a way of checking if other topics are using the same type, so memory allocated for `ddts` will hang around until the node destructor is called, even if it's not being used.